### PR TITLE
Debugger - python extension

### PR DESCRIPTION
**Pull request by abist**
Added code for python debugging scenario branched off of previous PR. Will set up demo with Ken and Vicky to see whether we want to take it or not.

### CODE REVIEW

File: extensions/integration-tests/package.json [hunk 1]
```diff
@@ -10,7 +10,6 @@
   },
   "activationEvents": [
 		"onFileSystem:memfs",
-    "onDebug",
```
Comment (abist) [inline extensions/integration-tests/package.json:13]: Not sure why this was added, but it doesn't seem useful at all. It only causes to have errors that prompt the user to install the postgres extension

**Comment by abist** [comment]
@kburtram Ping 😄

**Comment by Charles-Gagnon** [comment]
@abist Can this be closed for now?

**Comment by kburtram** [comment]
Closing older PRs.  Please reactivate if still needed.

**Pull request closed without merging**